src/nvim/edit.c

```diff
@@ -574,10 +574,8 @@ static int insert_check(VimState *state)
     foldCheckClose();
   }
 
-  int cmdchar_todo = s->cmdchar;
```

Comment:
Since cmdchar_todo was used only here, I deleted the variable itself, but I will return it if future Vim patch seems to be difficult to apply.

Reply:
yeah it looks useless, even in vim's source.